lib/adapters/QUnitAdapter.js

```diff
@@ -178,12 +179,23 @@ export default class QUnitAdapter extends EventEmitter {
 
   onTestStart (details) {
     this.errors = []
+    this.assertions = []
   }
 
   onLog (details) {
-    if (!details.result) {
-      this.errors.push(details)
+    var assertion = {
+      success: details.result,
+      actual: details.actual || null,
+      expected: details.expected || null,
+      message: details.message,
+      stack: details.source || undefined
```

Comment:
I agree with the ones above, but I think this is necessary, because passed assertion must have the stack undefined, at least this is what we discussed in #79